src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php

```diff
@@ -834,6 +834,11 @@ public function leaveHash(Cursor $cursor, $type, $class, $hasChild, $cut)
         parent::leaveHash($cursor, $type, $class, $hasChild, 0);
     }
 
+    public function dumpImage(Cursor $cursor, $str, $bin, $cut)
+    {
+        $this->line .= sprintf('<img src=data:image/png;base64,%s />', base64_encode($str));
```

Comment:
you should keep quotes around the attribute to avoid issues (unquoted attributes require additional escaping, and the trailing `=` chars may cause issues if unquoted)